### DISCUSSION TITLE
Welcome to abess Discussions!

### DISCUSSION THREAD

**Discussion by Mamba413** [abess-team/abess#300] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dududurant** [comment]
Hello,I'm a colledge student. I find the R package has an output "screening.vars" which can select out the variables of the maximum k values based on the SIS. What  I want is the order of the selected k variables. So how can I fix it without  understanding the algorithms?

**Reply by Mamba413** [reply]
You can use a for-loop to obtain the order of the screened variables. Here’s an example:

```r
library(abess)
n <- 100
p <- 100
support.size <- 3

dataset <- generate.data(n, p, support.size)
for (i in 1:10) {
  abess_fit <- abess(dataset[["x"]], dataset[["y"]], support.size = i,
                     screening.num = i, max.splicing.iter = 1)
  print(sprintf("Screen %s variable: ", i))
  print(abess_fit[["screening.vars"]])
}
```
